src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -79,6 +79,13 @@ public interface IPKAnalysesTask : OSPSuite.Core.Domain.Services.IPKAnalysesTask
       ///    Starts the calculation of the Auc DDI Ratio for all compounds /> by switching off all other applications
       /// </summary>
       void CalculateDDIRatioFor(Simulation simulation);
+
+      /// <summary>
+      ///    Returns true if global pk-analysis can be calculated for the given <paramref name="protocol" /> otherwise false
+      /// </summary>
+      /// <param name="protocol"></param>
+      /// <returns></returns>
+      bool CanCalculateGlobalPKFor(Protocol protocol);
```

Comment:
Seems to be the right place, where we calculate global pk, to wonder if we can calculate global pk. Doing this in the presenter was not cohesive.